neural_compressor_ort/quantization/quantize.py

```diff
@@ -12,79 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from pathlib import Path
+import pathlib
 from typing import Union
 
 import onnx
 from onnxruntime.quantization.quantize import QuantConfig
```

Comment:
update to `quantization.QuantConfig`?

Reply:
I see, how about import `quantize`?
```python
from onnxruntime.quantization import quantize as ort_quant
ort_quant.QuantConfig

```

Reply:
it is also can't work, onnxruntime has quantize.py in quantization folder and quantize.py contains quantize function.
```python
from onnxruntime.quantization import quantize as ort_quant
```
import quantize function and QuantConfig is recognized as an attribute, so it will raise error: AttributeError: 'function' object has no attribute 'QuantConfig'

Reply:
ok, let's leave it as is.